lib/webdriverio.js

```diff
@@ -230,6 +234,14 @@ let WebdriverIO = function (args, modifier) {
             })
     }
 
+    function mkRequestHanlderDecorator (opts) {
```

Comment:
It creates `RequestHandler` instance with updated `defaultOptions` leaving original `requestHandler` unchanged.